<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManagePet</class>
 <widget class="QWidget" name="ManagePet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton1">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>31000</width>
         <height>21</height>
        </size>
       </property>
       <property name="text">
        <string>â¬…</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>201</width>
         <height>27</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>MS Shell Dlg 2</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Pet Management</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton2">
       <property name="minimumSize">
        <size>
         <width>51</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>12000</width>
         <height>21</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Manjari</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"> QPushButton{ 

}</string>
       </property>
       <property name="text">
        <string>Logout</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="font">
        <font>
         <family>Manjari</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
	margin-right: 20%;
}</string>
       </property>
       <property name="text">
        <string>              Name</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="font">
        <font>
         <family>Manjari</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Species</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="font">
        <font>
         <family>Manjari</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Breed</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_10">
       <property name="font">
        <font>
         <family>Manjari</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QCheckBox" name="checkBox1">
       <property name="maximumSize">
        <size>
         <width>14</width>
         <height>14</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QCheckBox {
	margin-left: 0%;
	margin-right: 0%;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="image1">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="name1">
       <property name="font">
        <font>
         <family>Manjari</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Name</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="species1">
       <property name="font">
        <font>
         <family>Manjari</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Spc</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="breed1">
       <property name="font">
        <font>
         <family>Manjari</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Brd</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_12">
     <item>
      <widget class="QCheckBox" name="checkBox2">
       <property name="maximumSize">
        <size>
         <width>14</width>
         <height>14</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="image2">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="name2">
       <property name="font">
        <font>
         <family>Manjari</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Name</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="species2">
       <property name="font">
        <font>
         <family>Manjari</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Spc</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="breed2">
       <property name="font">
        <font>
         <family>Manjari</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Brd</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QCheckBox" name="checkBox3">
       <property name="maximumSize">
        <size>
         <width>14</width>
         <height>14</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="image3">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="name3">
       <property name="font">
        <font>
         <family>Manjari</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Name</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="species3">
       <property name="font">
        <font>
         <family>Manjari</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Spc</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="breed3">
       <property name="font">
        <font>
         <family>Manjari</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Brd</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="prev">
       <property name="font">
        <font>
         <family>Manjari</family>
        </font>
       </property>
       <property name="text">
        <string>Prev</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="page">
       <property name="font">
        <font>
         <family>Manjari</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Page 1 of X</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="next">
       <property name="font">
        <font>
         <family>Manjari</family>
        </font>
       </property>
       <property name="text">
        <string>Next</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="viewSelect">
         <property name="font">
          <font>
           <family>Manjari</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    border: 2px solid #8f8f91;
    border-radius: 40px;
    background-color: #fcf4a3;
    min-width: 80px;
	min-height: 80px;
}</string>
         </property>
         <property name="text">
          <string>View
 Selected</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="removeSelect">
         <property name="font">
          <font>
           <family>Manjari</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    border: 2px solid #8f8f91;
    border-radius: 40px;
    background-color: #69C4FF;
    min-width: 80px;
	min-height: 80px;
}</string>
         </property>
         <property name="text">
          <string>Remove
 Selected</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="updateSelect">
         <property name="font">
          <font>
           <family>Manjari</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    border: 2px solid #8f8f91;
    border-radius: 40px;
    background-color: #ea4c46;
    min-width: 80px;
	min-height: 80px;
}</string>
         </property>
         <property name="text">
          <string>Update
 Selected</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="addSingle">
         <property name="font">
          <font>
           <family>Manjari</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    border: 2px solid #8f8f91;
    border-radius: 40px;
    background-color: #3ded97;
    min-width: 80px;
	min-height: 80px;
}</string>
         </property>
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="addBulk">
         <property name="font">
          <font>
           <family>Manjari</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    border: 2px solid #8f8f91;
    border-radius: 40px;
    background-color: #E53FFF;
    min-width: 80px;
	min-height: 80px;
}</string>
         </property>
         <property name="text">
          <string>Add Bulk</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
